src/app/ecosystem-explorer/project-form/actions/submitProject.ts

```diff
@@ -2,7 +2,7 @@
 
 import slugify from 'slugify'
 
-import { ECOSYSTEM_PROJECTS_DIRECTORY_PATH } from '@/constants/paths'
+import { ECOSYSTEM_EXPLORER_DIRECTORY_PATH } from '@/ecosystem-explorer/utils/getEcosystemProjectData'
```

Comment:
Why did we change where we are importing this from?

Reply:
I followed the pattern we use in `getDigestArticleData` and `getEventsData` where we defined the DIRECTORY_PATH?

Reply:
I meant this part `from '@/ecosystem-explorer/utils/getEcosystemProjectData'` - Why not keep `'@/constants/paths'`?